inc/server/class-form-server.php

```diff
@@ -58,12 +60,34 @@ public function register_routes() {
 				),
 			)
 		);
+		register_rest_route(
+			$namespace,
+			'/forms',
+			array(
+				array(
+					'methods'             => \WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_form_settings' ),
+					'permission_callback' => function () {
+						return __return_true();
+					},
+				),
+			)
+		);
 	}
 
 	/**
-	 * Search WordPress Plugin
+	 * Get Form settings
 	 *
-	 * Search WordPress plugin using WordPress.org API.
+	 * @return mixed|\WP_REST_Response
+	 */
+	public function get_form_settings() {
```

Comment:
This can be accessed by anyone, even by a hacker without needing any permissions. Is it fine to expose site key? If yes then no need for it to be a rest route, it can simply be passed as a global variable using `wp_localize_script` instead.

Reply:
The site key is public. All you can do with it is to create the captcha field. 

Reference: https://developers.google.com/recaptcha/docs/v3 - you can put in the `div` or when you load the script or use it where you want.

Reply:
Yea, then you can just pass it to the script using wp_localize_script instead. Don't need to make any REST calls for this then.